packages/terra-form-input/src/InputField.jsx

```diff
@@ -115,14 +119,17 @@ const InputField = (props) => {
     isLabelHidden,
     label,
     labelAttrs,
+    maxWidth,
+    onChange,
     refCallback,
     required,
     showOptional,
-    onChange,
     value,
     ...customProps
   } = props;
 
+  const styles = 'style' in props ? props.style : { maxWidth: props.maxWidth };
```

Comment:
`maxWidth` will need to be merged into style properties if either one exists, otherwise all previous styles will be overwritten. 

```
<Input style={{ color: 'red' }} maxWidth="200px" />
```